db/mysql/ops.go

```diff
@@ -139,7 +145,7 @@ const (
 func (c *client) LoginUser(ctx context.Context, email string, password string) (string, error) {
```

Comment:
```suggestion
func (c *client) GetUserWithCredential(ctx context.Context, email string, password string) (string, error) {
```

Reply:
move token creation to service layer

Reply:
If you want to save a token in the database, only store the JTI (UUID ID of the token) in the database. During every token verification cycle, check if the JSON token ID exists in the database. In the logout API, remove the token's JTI from the database.